### PR TITLE
get data from push

### PR DESCRIPTION
**Pull request by Mr-Hqq**

### CODE REVIEW

File: android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java [hunk 1]
```diff
@@ -133,6 +135,30 @@ public void run() {
         });
     }
```
Comment (Mr-Hqq) [inline android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java]: Convert JSONObject to WritableMap For Put in response wriatblemap

File: android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java [hunk 2]
```diff
@@ -133,6 +135,30 @@ public void run() {
...
+        Iterator iter = json.keys();
+        while (iter.hasNext()) {
+            String key = iter.next();
+            try {
+                if (json.get(key) instanceof Integer) {
+                    response.putInt(key, (Integer) json.get(key));
+                } else if (json.get(key) instanceof String) {
+                    response.putString(key, (String) json.get(key));
+                } else if (json.get(key) instanceof JSONObject) {
+                    toWritableMap((JSONObject) json.get(key));
```
[the remark this answers is not in the document]
Comment (Mr-Hqq) [reply android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java]: no need to change.
its just Recursive function for JSONObject variable to return to funtion for any variable to separate by any type.
Or can be your code to add as writeableMap.

[the remark this answers is not in the document]
Comment (behrad) [reply android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java]: Check this please @Mr-Hqq

**Comment by behrad** [comment]
Re-create your PR to `dev` or any fix branch @Mr-Hqq 
We don't merge into master. Closing this.

File: android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (behrad) [reply android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java]: @amir-yaghoobi Please test for any kind of data objects when merged into DEV branch

**Pull request closed without merging**